src/app/toolbar/components/dialog-date/dialog-date.component.html

```diff
@@ -0,0 +1,11 @@
+<div mat-dialog-content>Enter date of raport</div>
```

Comment:
Mat dialog content is not a title. It is meant for making content of dialog scrollable. Mat form field should be inside dialog content